solidity/test/token_grant/TestTokenGrantStake.js

```diff
@@ -287,20 +287,36 @@ describe('TokenGrant/Stake', function() {
   })
 
   it("should not allow to delegate to the same operator after recovering stake", async () => {
-    let tx = await delegate(grantee, operatorOne, grantAmount)
+    let tx = await delegate(grantee, operatorOne, minimumStake)
     let createdAt = web3.utils.toBN((await web3.eth.getBlock(tx.receipt.blockNumber)).timestamp)
     await time.increaseTo(createdAt.add(initializationPeriod).addn(1))
     tx = await grantContract.undelegate(operatorOne, {from: grantee})
     await time.increaseTo(createdAt.add(grantUnlockingDuration))
     await grantContract.recoverStake(operatorOne, {from: grantee});
-    await stakingEscrowContract.withdraw(operatorOne, {from: grantee});
 
     await expectRevert(
-      delegateLiquid(grantee, operatorOne, minimumStake),
+      delegate(grantee, operatorOne, minimumStake),
```

Comment:
Don't we want to leave also the previous case where we tried to top up with liquid tokens?

Reply:
This scenario is covered in `TestTopUps.js` by two `""can not be done from liquid tokens""` test cases.